core/gui/src/app/workspace/component/left-panel/left-panel.component.ts

```diff
@@ -95,4 +100,37 @@ export class LeftPanelComponent implements OnDestroy, OnInit {
       this.height = height!;
     });
   }
+
+  resetPanelPosition() {
+    this.dragPosition = { x: this.returnPosition.x, y: this.returnPosition.y };
+  }
+
+  parseTranslate3d(translate3d: string): [number, number, number] {
```

Comment:
Can we put this function in a separate file instead of duplicate this function 3 times?